functions/exercise4/scriptSpect.js

```diff
@@ -0,0 +1,8 @@
+/// <reference path="../../typings/globals/jasmine/index.d.ts" />
+/*globals validatePhrase*/
+
+describe("Check if", function () {
+  it("phrase is number ", function () {
+    expect(function () { validatePhrase(Number); }).toThrow(new Error("Phrase can't be number"));
```

Comment:
Is "Phrase can't be number" appropriate message here?
It is gramatically incorrect (it should be "a number").
Apart from that it would be better to describe what you expect there instead of what you do not expect.